kratos/geometries/geometry.h

```diff
@@ -2679,23 +2887,22 @@ class Geometry : public PointerVector<TPointType>
 
     GeometryData const* mpGeometryData;
 
+    Kratos::shared_ptr<PointsArrayType> mpPointVector;
```

Comment:
@pooyan-dadvand 
@tteschemacher told me that you agreed here to use a shared ptr